src/types.ts

```diff
@@ -69,19 +107,27 @@ export enum IntegrationName {
   SIFT_SCORE = "SIFT SCORE",
   FINGERPRINTJS = "FINGERPRINTJS",
   STRIPE_IDENTITY = "STRIPE_IDENTITY",
-  TWILIO = "TWILIO",
+  MFA_TWILIO = "MFA_TWILIO"
```

Comment:
If the TWILIO integration code does not need to run on the client-side (see my previous comment in that integration file), then we can simplify this by just calling it "MFA".